modules/oc-mirror-imageset-config-params.adoc

```diff
@@ -269,3 +269,12 @@ channels:
 |Boolean. The default value is `false`.
 
 |===
+
+[NOTE]
+====
+In certain cases error occurs when the `minVersion` and `maxVersion` properties are used to filter for a specific version range for an operator in the `ImageSetConfiguration` parameters. The error message will state that there are `multiple channel heads`. This is because when the filter is applied, the update graph of the operator is truncated.
+
+The Operator Lifecycle Manager in OpenShift, however, expects that every operator channel contains versions that form an update graph with exactly one end point, the latest version of the operator. When applying the filter range that graph can turn into two or more separate graphs or a graph that has more than one end point.
+
+To avoid this error, do not filter out the latest version of an operator. If you still run into the error, depending on the operator, either the `maxVersion` property needs to be increased or the `minVersion` property needs to be decreased. Because every operator graph can be different, you may need to adjust these values step-wise until the error is gone.
+====
```

Comment:
step-wise means to follow the step-by-step procedure until the error is gone.

Reply:
Changed the sentence to "Because every operator graph can be different, you might need to adjust these values, according to the procedure, until the error is gone." Can you check if this looks good?

Reply:
@subhtk Perfect! Thanks!